tests/regression/44-trier_analyzer/27-rec4.c

```diff
@@ -12,7 +12,9 @@ void rec (int **u, int i) {
   else {
     i++;
     u = (int **)malloc(sizeof(int *));
+    int** v = u;
     rec(u, i);
+    assert(v == u); //UNKNOWN
```

Comment:
Should this also be `UNKNOWN!`?

Reply:
Why?
`u` is a local variable right, so the formal `u` of the callee is assigned the pointer the caller allocated. 
In the recursive call, `u` is assigned a new value (a fresh pointer in this case), but that is a simple assignment to a local variable that does not affect the value in the caller. 
The case where it is unknown would be if `*u` was assigned. Or am I missing something?

Reply:
I didn't actually investigate the program, but am just wondering about the clarity of our annotations going forward. It should either be `UNKNOWN!` if it's unknown in concrete, `TODO` if it's something we might want to handle in the future or `UNKNOWN (explanation)` if it's intended to be less precise than ideal. A simple `UNKNOWN` is ambiguous about the intent.

Reply:
We still have `//UNKNOWN` all over our tests, but I guess I can change it to `//TODO` here.